tools/influx_si/influx_si.xml

```diff
@@ -0,0 +1,91 @@
+<tool id="influx_si" name="influx_si" version="0.8.0+galaxy0" python_template_version="3.5">
+    <description>metabolic flux analysis</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements" />
+    <!--
+#silent sys.stderr.write("!!!! Cheetah Template Variables !!!!\n")
+#for k,v in $searchList[2].items()
+#silent   sys.stderr.write(" %s = %s\n" % (str(k), str(v) ))
+#end for
+    -->
+    <version_command>influx_s --version</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+       #set $inp_all = [$input_main, ($prlexp.input_aux if $prlexp.input_aux else [])]
+       #if $si.s_i == 'i':
+         #silent $inp_all.append($si.input_ti)
+       #end if
+
+       mkdir outdir && cd outdir;
```

Comment:
can you please replace all ´;´ with `&&`

Reply:
I replaced only `;` ending lines. Those required by `for` loop syntax were left.